src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/uploaddocuments/UploadDocumentFurtherEvidenceHandler.java

```diff
@@ -50,14 +45,9 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             response.addError("You need to provide a file and a document type");
             return response;
         }
-        if (!isFileUploadedAPdf(caseData.getDraftSscsFurtherEvidenceDocument())) {
```

Comment:
This error response is still valid when `feature.upload-audio-video-evidence.enabled` is false

Reply:
As we have restriction ccd-def now, it is not possible to upload non pdf file isn't it?